index.html

```diff
@@ -69,12 +69,18 @@ <h1><a href="#intro">Hacktober<span>fest</span>NH</a></h1>
           <!-- <li><a href="#hotels">Hotéis</a></li> -->
           <!-- <li><a href="#gallery">Galeria</a></li> -->
           <li><a href="#supporters">Patrocínio</a></li>
+          <li>
+            <a class="dropdown-toggle" href="javascript:void(0);" data-toggle="dropdown">Edições anteriores</a>
+            <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
+              <a class="dropdown-item" href="./past_events/2019/index.html">2019</a>
+            </div>
+          </li>
```

Comment:
Can we change the dropdown background color?

![image](https://user-images.githubusercontent.com/1295197/66585409-25ca1f80-eb5d-11e9-8dcf-902f686ee430.png)

Reply:
Hi @paulodiovani 

The background-color was changed.

![IMG_2039](https://user-images.githubusercontent.com/5265969/66707759-7e93e680-ed1c-11e9-8794-24f9816058e9.jpg)